### PR TITLE
GitHub container registry typos

### PR DESCRIPTION
**Pull request by aegilops**
Typos of `ghcr.io`.

Tested with my `test.py` script.

I did GitHub code searches to confirm that people make all of these mistakes.

`gcr.io` (dropping the h), `hcr.io` (dropping the g) are both different container registries.

`hrc.io` - is a political org. It's unlikely to be intentionally in Enterprise code, but I didn't include it.

### CODE REVIEW

**Comment by aegilops** [comment]
I need to generate the README.md for this one

File: uri/patterns.yml [hunk 1]
```diff
@@ -90,3 +90,20 @@ patterns:
       - name: non-english.txt
         start_offset: 19
         end_offset: 25
+
+
+  - name: GitHub Container Registry typos
```
Comment (GeekMasher) [inline uri/patterns.yml:139]: @aegilops Can you run the Markdown generator so this is added in the README?

**Review by GeekMasher — COMMENTED** [review]
Once you have the markdown, we can merge

File: uri/patterns.yml (hunk 1 above)
Comment (aegilops) [reply uri/patterns.yml:139]: On it

Comment (aegilops) [reply uri/patterns.yml:139]: - Merged in changes from main
- generated README.md for everything

**Review by GeekMasher — APPROVED** [review]
Perfect 🚀

**Pull request merged**